src/test/java/org/folio/support/DatabaseHelper.java

```diff
@@ -99,6 +100,18 @@ public Integer queryAuthoritySourceFileSequenceCurrentValue(String tenant, UUID
     return jdbcTemplate.queryForObject(sql, Integer.class);
   }
 
+  public void disableAutoLinking(String tenant, Integer ruleId) {
+    var sql = "UPDATE " + getDbPath(tenant, INSTANCE_AUTHORITY_LINKING_RULE)
+        + " SET auto_linking_enabled = ? where id = ?";
+    jdbcTemplate.update(sql, false, ruleId);
+  }
+
+  public void enableAutoLinking(String tenant, Integer ruleId) {
+    var sql = "UPDATE " + getDbPath(tenant, INSTANCE_AUTHORITY_LINKING_RULE)
+        + " SET auto_linking_enabled = ? where id = ?";
+    jdbcTemplate.update(sql, true, ruleId);
+  }
+
```

Comment:
Maybe better create one method like 'setAutoLinking' and put enabling value as parameter to avoid duplication.